docs/UserGuide.md

```diff
@@ -206,117 +174,120 @@ Date:   Fri Feb 9 19:13:13 2018 +0800
-Each light blue bar represents the contribution timeline of an individual author for a specific repository. Each row, there are several **ramps**.
-- The area of the ramp is proportional to the amount of contribution the author did at that time period.
-- The position of the right edge of the ramp (perpendicular to the blue bar) is dependent on the time period that ramp represents.
-- To make comparison between two authors easier, the color of the ramps that represent different authors' contributions at the same time period are the same.
-- The timelines (blue bar) of the charts should be aligned, so that the comparison of contribution between two authors are easier.
-- There is no limit to the area of the ramp. If the contribution for a time period is too large compared to the rest of the time period, it is going to overlap with the neighbor ramps. Thus, the ramps are transparent, so they will not cover their neighbors.
-- When user hovers the pointer over a ramp, as shown in the above Figure, total amount of contribution over the time period will be shown.
-- Clicking on the ramp will redirect user to the GitHub page, which contains all the commits within the fixed time period.
+`repo-config.csv` file contains repo-level config data as follows:
 
-#### Contribution Bar
-The total amount of code contributed is represented by the **red bars**, and the length of these red bars is proportional to the total contribution of the corresponding author.
-Hovering over the bar shows the exact amount of contribution.
-If the author contributes **too much** compared to other authors, there will be multiple red bars in his `Chart Panel`.
+* First row: column headings, ignored by RepoSense
+* Second row: repo data
+
+Here is an example:
+
+Repository's Location|Branch|File formats|Ignore Glob List|Ignore standalone config|Ignore Commits List
+---------------------|------|------------|----------------|------------------------|-------------------
+`https://github.com/foo/bar.git`|`master`|`java;css`|`test/**`|`yes`|`2fb6b9b2dd9fa40bf0f9815da2cb0ae8731436c7;c5a6dc774e22099cd9ddeb0faff1e75f9cf4f151`
+
+<br>
+
+Column Name | Explanation
+----------- | -----------
+Repository's Location | The `GitHub URL` or `Disk Path` to the git repository e.g., `https://github.com/foo/bar.git` or `C:\Users\user\Desktop\GitHub\foo\bar`
+Branch | The branch to analyze in the target repository e.g., `master`. Default: the default branch of the repo
+[Optional] File formats<sup>*</sup> | The file extensions to analyze. Default: `adoc;cs;css;fxml;gradle;html;java;js;json;jsp;md;py;tag;xml`
+[Optional] Ignore Glob List<sup>*</sup> | The list of file path globs to ignore during analysis for each author. e.g., `test/**;temp/**`
+[Optional] Ignore standalone config | To ignore the standalone config file (if any) in target repository, enter **`yes`**. If the cell is empty, the standalone config file in the repo will take precedence over configurations provided in the csv files.
```

Comment:
We can mention here that by default, the `Standalone Config` will overwrite this `config.csv` settings, and this setting is to prevent that overwriting.

Reply:
Isn't the second sentence mean the same thing? Or is there another scenario I'm missing?

Reply:
Whoops, missed the second sentence here. 
Yes, the second sentence does mean the same thing I meant.